src/components/Header.js

```diff
@@ -23,6 +23,11 @@ class Header extends Component {
 		collapseMenu: PropTypes.bool.isRequired,
 	}
 
+	formatDuration(duration) {
```

Comment:
Probably worth moving to `utilities` to keep implementation consolidated.

Reply:
I c/p it from some other module since I couldn't access it in header

Reply:
Exactly